README.md

```diff
@@ -6,33 +6,31 @@
 
 Follow the project progress at: [ETC Block Explorer Development](https://github.com/ethereumclassic/explorer)
 
-## Local installation
+## Requirements
 
-Clone the repo
+| Dependency  | Mac | Linux |
+|-------------|-----|-------|
+| [Node.js 8.x](https://nodejs.org/en/) | `brew install node` | [Node.js Install Example](https://nodejs.org/en/download/package-manager/) |
+| [MongoDB 4.x](https://www.mongodb.com) | `brew install mongodb` | [MongoDB Install Example](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/) |
 
-`git clone https://github.com/ethereumclassic/explorer`
+## Build and Run
 
-Download [Nodejs and npm](https://docs.npmjs.com/getting-started/installing-node "Nodejs install") if you don't have them
+  1. Clone the repository.
+  `git clone https://github.com/ethereumclassic/explorer`
 
-Install dependencies:
+  2. Go to the explorer subdirectory.
+  `cd explorer`
 
-`npm install`
+  3. Set up default configurations.
+  `cp config.example.json config.json`
 
-Install mongodb:
+  4. Install Node.js dependencies.
+  `npm install`
```

Comment:
just curious, why not use the `docker-compose up` ?

Reply:
Well let's leave docker as an option 😂 ( I am still using local installation without docker deployment )